packages/test/assembly/__tests__/index.spec.ts

```diff
@@ -411,3 +417,13 @@ function testCallbacks(): void {
   let b = ASON.deserialize<() => void>(buffer);
   b();
 }
+
+function testVirtual(): void {
+  let child = new Child();
+  child.a = 42;
+  let buffer = ASON.serialize(child);
+  let result = ASON.deserialize<Base>(buffer);
```

Comment:
This is the behavior that would have previously caused an error, however, this test now passes.